### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by Ana06**
Document supported Linux distributions, improve cloning capa source code documentation and issue templates. :bowtie:

### CODE REVIEW

File: doc/installation.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ We use PyInstaller to create these packages.
 
 The capa [README](../README.md#download) also links to nightly builds of standalone binaries from the latest development branch.
 
+### Linux Standalone installation
+
+The Linux Standalone binary has been built using GLIB 2.26.
+Consecuently it works when using GLIB >= 2.26.
```
Comment (williballenthin) [inline doc/installation.md:14]: "Consequently"

File: doc/installation.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ We use PyInstaller to create these packages.
...
+This requirement is satisfies by default in most newer distribution such as Ubuntu >= 18, Debian >= 10, openSUSE >= 15.1 and CentOS >= 8.
```
Comment (williballenthin) [inline doc/installation.md:15]: "is satisfied by default"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -3,24 +3,33 @@ name: Feature request
 about: Suggest an idea for capa
 
 ---
-
 <!--
+# Is your issue related to capa rules (for example an idea for a new rule)?
```
Comment (mr-tz) [inline .github/ISSUE_TEMPLATE/feature_request.md:7]: 👍

File: doc/installation.md [hunk 4]
```diff
@@ -29,31 +36,25 @@ You can now import the `capa` module from a Python script or use the IDA Pro plu
 If you'd like to review and modify the capa source code, you'll need to check it out from GitHub and install it locally. By following these instructions, you'll maintain a local directory of source code that you can modify and run easily. 
 
 ### 1. Check out source code
-Next, clone the capa git repository. We use submodules to separate code, rules, and test data. See below to get all data at once. To only get the source code and our provided rules (common), follow these steps:
+Next, clone the capa git repository.
+We use submodules to separate [code](https://github.com/fireeye/capa), [rules](https://github.com/fireeye/capa-rules), and [test data](https://github.com/fireeye/capa-testfiles).
```
Comment (mr-tz) [inline doc/installation.md:40]: 👍

**Pull request merged**